### PR TITLE
CVS output:

### PR DESCRIPTION
**Pull request by pburkholder**
## Changes proposed in this pull request:

- Give names to blanks to help track fields
- Add required KEV

-
-
-

## security considerations
These are safe values, and we'll overwrite the KEV should it actually come to pass that we have a KEV unpatched

### CODE REVIEW

**Review by bengerman13 — APPROVED** [review]

File: parse-nessus-xml.py [hunk 1]
```diff
@@ -213,14 +213,21 @@ def remediation_plan(vuln):
...
         weakness_name=vuln_report[vuln]["plugin_name"]
-        csvwriter.writerow(["CGXX","RA-5", weakness_name, "", "Nessus Scan Report", 
+        csvwriter.writerow(["CGXX","RA-5", weakness_name, weakness_desc, "Nessus Scan Report", 
             vuln_report[vuln]["id"], str(number_of_affected_hosts) + " production hosts", 
-            owner, "None", remediation_plan(weakness_name), start_date.date(), "", "Resolve", "", mmddYY, "Yes", mmddYY,
-            "CloudFoundry stemcell", risk_factor, risk_factor, "No", "No", "No" ])
+            owner, "None", remediation_plan(weakness_name), start_date.date(), sched_completion_date, 
+            "Resolve", milestone, mmddYY, "Yes", mmddYY,
+            "CloudFoundry stemcell", risk_factor, risk_factor, "No", "No", "No",
+            deviation_rationale, supporting_docs, comments, auto_approve, KEV])
```
Comment (bengerman13) [inline parse-nessus-xml.py:233]: this is getting hard to parse in my brain.
These names feel like a step in the right direction, but I'm still having a hard time holding enough of this list in context long enough to compare the diff

Comment (ccostino) [reply parse-nessus-xml.py:233]: Would just splitting it to multiple lines (one entry per line) help?

Comment (pburkholder) [reply parse-nessus-xml.py:233]: Next iteration (if any) I'll make it one entry per line.
